modules/azure/function_app_linux_managed_identity/outputs.tf

```
@@ -14,6 +14,10 @@ output "aad_application_name" {
   value = var.managed_identity_provider.existing == null ? "api://${var.managed_identity_provider.create.application_name}" : null
 }
 
-output "api_audience" {
+output "api_audiences" {
   value = local.allowed_audiences
 }
+
+output "api_audience" {
+  value = "api://${var.managed_identity_provider.create.application_name}"
```

```suggestion
  value = var.managed_identity_provider.create != null ? "api://${var.managed_identity_provider.create.application_name}" : null
```
This does not work because `create` could be null

---

But that would also mean that this output would become the same as `aad_application_name`, so it should be deleted

---

@tom-reinders its updated based on suggestion